### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by borekon**
Added usage and sample script

### CODE REVIEW

**Comment by XhmikosR** [comment]
The thing is that this info does not really belong here. This repo just provides Windows builds, so this info belongs upstream.

The samples script is something that we could add, though, since it's Windows specific. But please fix the indentation and use the proper snippet highlighting language.

File: README.md [hunk 1]
```diff
@@ -30,57 +30,13 @@ Currently only 64-bit (x64) is targeted.
...
-  --all-progressive force all output files to be progressive
-  --stdout          send output to standard output (instead of a file)
-  --stdin           read input from standard input (instead of a file)
-  ```
 ## Sample script
 
 There's a sample script that loop all the subfolders in a folder optimizing pictures.
 
-```
+```bash
```
Comment (XhmikosR) [inline README.md]: Should probably be `.bat`. Also please remove `pause` and lowercase any keywords

**Comment by XhmikosR** [comment]
How about adding an alternative script too?

```bat
for /f "tokens=*" %%i in ('dir /a:d /b') do (
  jpegoptim.exe --preserve --strip-all --totals --verbose "%%i\*.jpg"
)

rem or

for /f "delims=" %%i in ('dir /b /s *.jpg') do (
  jpegoptim.exe --preserve --strip-all --verbose "%%i"
)
```

**Comment by borekon** [comment]
Looks great too. How many levels it loops? Mine just 1 subfolder

On Feb 6, 2020 14:22, XhmikosR <notifications@github.com> wrote:

How about adding an alternative script too?

for /f "tokens=*" %%i in ('dir /a:d /b') do (
  jpegoptim.exe --preserve --strip-all --totals --verbose "%%i\*.jpg"
)

rem or

for /f "delims=" %%i in ('dir /b /s *.jpg') do (
  jpegoptim.exe --preserve --strip-all --verbose "%%i"
)

**Comment by XhmikosR** [comment]
I assume all.

**Comment by borekon** [comment]
Ok, put both and let the people choose.

**Comment by XhmikosR** [comment]
I'll take care of this tomorrow or so

**Pull request merged**